src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/VanillaMachinesListener.java

```diff
@@ -97,6 +97,25 @@ public void onAnvil(InventoryClickEvent e) {
         }
     }
 
+    @EventHandler(ignoreCancelled = true)
+    public void onCartographyTable(InventoryClickEvent e) {
+        // The Cartography Table was only ever added in MC 1.14
+        MinecraftVersion minecraftVersion = SlimefunPlugin.getMinecraftVersion();
+        if (!minecraftVersion.isAtLeast(MinecraftVersion.MINECRAFT_1_14)) {
```

Comment:
```suggestion
        if (!SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_14)) {
```
This should suffice.